docs/src/components/InlineFilter.tsx

```diff
@@ -0,0 +1,21 @@
+import { Fragment } from 'react';
+import { FilterChildren } from './FilterChildren';
+
+type InlineFilterProps = {
+  children: React.ReactNode;
+  filters: string[];
+};
+
+export const InlineFilter = ({ filters, children }: InlineFilterProps) => {
+  if (!filters || !Array.isArray(filters) || filters.length < 1) {
+    return <></>;
+  }
+
+  const filteredChildren: Array<React.JSX.Element> = [];
+
+  filters.forEach((filter) => {
+    filteredChildren.push(<Fragment key={filter}>{children}</Fragment>);
```

Comment:
I don't think I get this, if `filters` is an array, this rendering _all_ the children n times where n=filters.length. So if I have `<InlineFilter filters=["react","react-native"]>` the entire contents would rendered for both, which I don't think thats what we want right?